src/store/components/CollectionItemCard/index.tsx

```diff
@@ -0,0 +1,120 @@
+import React from 'react';
+
+import './style.less';
+import { useHistory } from 'react-router';
+
+type Uuid = string;
```

Comment:
Types should go in `src/types.ts` for now. If we get too many, we can eventually split them into the existing folder structure.